recipes/exactextract/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "exactextract" %}
+{% set version = "0.2.0.dev0" %}
```

Comment:
This one falls under https://github.com/conda-forge/cfep/blob/main/cfep-05.md

Reply:
Thank you for the review @ocefpaf , this is the only release made so far on pypi, so I wasn't sure about this.  I think I have addressed it.